core/src/main/scala/org/apache/spark/internal/config/package.scala

```diff
@@ -896,11 +896,7 @@ package object config {
 
   private[spark] val MAX_REMOTE_BLOCK_SIZE_FETCH_TO_MEM =
```

Comment:
shall we move it to `SparkConf.deprecatedConfigs`?

Reply:
Thansk, done in 2d2637b.